vector/src/main/java/im/vector/adapters/RoomViewHolder.java

```diff
@@ -302,4 +384,112 @@ public void onClick(View v) {
             }
         }
     }
+
+
+    /**
+     * Change the tag of the given room with the provided one
+     *
+     * @param roomId
+     * @param newTagOrder
+     * @param newTag
+     */
+    private void updateTag(final MXSession session, final String roomId, Double newTagOrder, final String newTag) {
+        //TODO Swipe : handle errors
+
+        RoomUtils.updateRoomTag(session, roomId, newTagOrder, newTag, new ApiCallback<Void>() {
```

Comment:
I think we should use here `SimpleApiCallback` class (instead of ApiCallback) to handle errors